examples/StandardFirmataBLE/bleConfig.h

```diff
@@ -106,6 +106,15 @@ BLEStream stream;
 // BLEStream stream(BLE_REQ, BLE_RDY, BLE_RST);
 // #endif
 
+/*
+ * Adafruit Bluefruit nRF52 Boards
+ */
+#ifdef ARDUINO_NRF52_ADAFRUIT
+#include "utility/BluefruitStream.h"
+BLEStream stream;
+#endif
```

Comment:
Sorry for late response, this is the pull request that contains only Boards.h for adafruit nrf52 boards series. Thank you very much
https://github.com/firmata/arduino/pull/381